app/views/admin/conferences/dashboard/_activity_item.html.erb

```diff
@@ -13,7 +13,7 @@
                 <p class="nm">
                   <%= t("activities.actions.#{activity.action}",
                         user: "<span class=semibold>#{activity.user.display_name}</span>",
-                        speech: link_to(activity.subject.topic.subject, [activity.conference, activity.subject.commentable])
+                        speech: link_to(activity.subject.topic.subject, ["admin", activity.subject.commentable])
```

Comment:
@xaph 
:admin instead of :admin ? 

![hayırdır](http://www.capsomat.com/media/created/65kym7.jpg)